Tensile/SolutionStructs.py

```diff
@@ -2870,12 +2870,18 @@ def assignDerivedParameters(state):
         if state["LocalReadVectorWidth"] != state["VectorWidth"]:
           reject(state, "not support LRVW != VW with nonMI kernel")
 
+    optPad = state["LocalReadVectorWidth"]
+    readWidth = state["LocalReadVectorWidth"]*state["ProblemType"]["DataType"].numBytes()//4
+    if state["EnableMatrixInstruction"]:
+      if state["MatrixInstB"] == 1 and state["MatrixInstM"] == 16 and \
+          (readWidth == 4 or readWidth == 1):
+        optPad *= 2
```

Comment:
Maybe a bit of comments describing how conflict-free padding came into being (without going too explicit on the idiosyncrasies of the LDS hardware) is also appreciated.

Reply:
add more comment